share/governance.md

```diff
@@ -0,0 +1,111 @@
+
+
+
+Contributors
+------------
+
+*How to become one*: Be a collaborator and be nominated to the PMC as a contributor. Nominations should be sent to <email address for nominations>. You may nominate yourself. It is also common to give contributor status to an individual who donates code to the project by migrating a repository to the github namespace.
+
+Contributors are collaborators who have shown wide dedication to the Vox Pupuli project in general or deep dedication to one project in particular, and the ability to work well with collaborators, users, and other contributors. The contributors have responsibilities beyond the collaborators. In particular, contributors formally decide on whether a pull request is merged or a release is cut. Contributors are recognized and valued for their contributions, in whatever form those contributions take. Tasks such as communication, documentation, art, packaging, and testing are all valued contributions. A contributor will use lazy consensus to decide on whether to merge a pull request from a collaborator. If the discussion is no longer moving towards a consensus, the PMC must vote via lazy consensus on whether the patch should be applied.
+
+
+
+Project Management Committee (PMC)
+----------------------------------
+
+*How to become one*: Win in the yearly election.
+
+
+The project management committee has responsibilities beyond contributors that include participating in strategic planning, release planning and approving changes to the governance model. One of the most important duties is to uphold the community code of [conduct](https://voxpupuli.org/coc/) and ensue its values. The PMC has to make decisions when community consensus cannot be reached.
+
+The PMC has final say over who can become a committer and will use lazy consensus for approval. Discussion over committer nominations will be done in private.
+
+Membership of the PMC is by election. [Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) voting is held once a year. A non-running election official is appointed by the PMC and runs the election. There are 5 members of the PMC. Terms are 1 year.
+
+The PMC doesn't have specific roles or a chairperson.
+
+
+
+Specialized Roles
+-----------------
+
+
+There are three officer roles in VoxPupuli. They can all be held by the same person (though this isn't ideal), as long as the work is getting done. They can have any other role in the organization, including being on the PMC. The PMC is responsible for requesting volunteers and appointing a volunteer to the role. Appointments last until either the officer steps down, or the unlikely and unfortunate situation where the PMC removes the officer.. The PMC can add more officer roles as it sees fit by modifying this document.
+
+1) Election Officer: The election officer can be on the PMC, as long as they are not running in the election they are officiating
```

Comment:
It means that the election officer would not be one of the PMC candidates if the entire election for PMCs is run at once. It may not happen at once, in case someone is removed.

I think the idea is that PMC is separate from officers.